### PR TITLE
Upgrade Svelte and Sveltekit

### PR DESCRIPTION
**Pull request by Jakeii**
## What does this change?
Upgrade Svelte from v3 to v5 and Sveltekit 1.0.0-next.394 to 2.0.0

We were on an unstable beta version this upgrade to a stable version is long overdue!

The most significant API chages isThe new routing that came with Sveltekit 1.0 https://github.com/sveltejs/kit/discussions/5774, which means

- Different file structure for routes
- no multiple script tags with `context="module"`

Due to the large changes, probably best to wait for any active migrations to finish

### CODE REVIEW

File: src/templates/ssr/interscroller/index.svelte [hunk 1]
```diff
@@ -6,14 +6,17 @@
 	<img
 		src="[%TrackingPixel%]"
 		class="creative__pixel creative__pixel--displayNone"
+		alt="pixel"
```
Comment (domlander) [inline src/templates/ssr/interscroller/index.svelte:9]: I don't think we should have an `alt` tag for these `img`'s. Tracking pixel's aren't displayed to the user so we should probably hide them from screen readers too.

File: src/templates/components/fonts/Egyptian.css [hunk 2]
```diff
@@ -1,6 +1,7 @@
 @font-face {
 	font-family: 'GuardianTextEgyptian';
-	src: url('https://assets.guim.co.uk/static/frontend/fonts/guardian-textegyptian/noalts-not-hinted/GuardianTextEgyptian-Regular.woff2')
+	src:
+		url('https://assets.guim.co.uk/static/frontend/fonts/guardian-textegyptian/noalts-not-hinted/GuardianTextEgyptian-Regular.woff2')
 			format('woff2'),
 		url('https://assets.guim.co.uk/static/frontend/fonts/guardian-textegyptian/noalts-not-hinted/GuardianTextEgyptian-Regular.woff')
```
Comment (domlander) [inline src/templates/components/fonts/Egyptian.css:6]: I can't tell what the changes are in these files?

File: src/templates/components/PaidForHeader.svelte [hunk 3]
```diff
@@ -22,10 +18,7 @@
 	<div class="{templateType}-paid-content-and-title">
 		<div class="paid {templateType}-paid-content">
 			<strong>Paid content</strong>
-			<Button
-				aria-controls="popup"
```
Comment (domlander) [inline src/templates/components/PaidForHeader.svelte:26]: Why are we losing `aria-controls` and `aria-expanded`?

File: src/templates/ssr/interscroller/index.svelte (hunk 1 above)
Comment (Jakeii) [reply src/templates/ssr/interscroller/index.svelte:9]: good point!

File: src/templates/components/fonts/Egyptian.css (hunk 2 above)
Comment (Jakeii) [reply src/templates/components/fonts/Egyptian.css:6]: the linter has added newlines, now using guardian prettier settings

File: src/templates/components/PaidForHeader.svelte (hunk 3 above)
Comment (Jakeii) [reply src/templates/components/PaidForHeader.svelte:26]: redundant as they're already on the `Button` component! https://github.com/guardian/commercial-templates/blob/jlk/upgrade-sveltekit/src/templates/components/Button.svelte#L1

**Review by domlander — APPROVED** [review]

File: src/templates/ssr/interscroller/index.svelte (hunk 1 above)
Comment (domlander) [reply src/templates/ssr/interscroller/index.svelte]: I like the `aria-hidden` addition 👍

**Pull request merged**